packages/docs/src/pages/en/components/navigation-drawers.md

```diff
@@ -87,6 +81,18 @@ When using the **rail** prop, the drawer will shrink (default 56px) and hide eve
 
 <ExamplesExample file="v-navigation-drawer/prop-rail-variant" />
 
+#### Expand on hover
+
+Places the component in **rail** mode and expands once hovered. This **does not** alter the content area of **v-main**. The width can be controlled with the **rail-width** property.
+
+<ExamplesExample file="v-navigation-drawer/prop-expand-on-hover" />
+
+#### Rail width
+
+The drawer will resize to the value of **rail-width** (or default to `56px` if undefined) and will hide all **v-list** items except the first. This must be used in **rail** mode.
```

Comment:
You don't have to change this, but putting default values in content is just one more place we have to remember to update if it ever changes.